### ISSUE TITLE
Projects created before pr#540 need updated format for `vernacularWritingSystem` and `analysisWritingSystems` properties

### ISSUE THREAD

**Issue by imnasnainaec** [sillsdev/TheCombine#557]
Pull request #540 introduces a more complex `writingSystem` type. This affects two properties: `vernacularWritingSystem` and `analysisWritingSystems`.

Anyone with admin access to an active Mongo database for TheCombine can run the following code from command-line to add default values in the correct format to the affected properties. **This will overwrite any existing values on those properties!**

`mongo -eval "db.getSiblingDB('CombineDatabase').getCollection('ProjectsCollection').update({vernacularWritingSystem: {\$type: 'string'}}, {\$set: {'vernacularWritingSystem': {Name:'',Bcp47:'',Font:''}, 'analysisWritingSystems': [{Name:'English',Bcp47:'en',Font:''}]}}, {multi: true})"`

**Comment by jmgrady** [comment]
I updated thecombine.languagetechnology.org's CombineDatabase.  I used the attached script to maintain the current values for `vernacularWritingSystem` rather than overwrite to the default of English.  The script is limited in that it is based on the data that were in the database at the time of the migration.  In particular, the hash `%langNames`, only has a mapping from the IANA sub-tags to language name for the values that were found in the data at the time:
```
my %langNames = (
	"es" => "Spanish",
	"bum" => "Bulu (Cameroon)",
	"qaa-x-stc-natqgu" => "Natqgu"
);
```
This should be extended to use on other databases.
[migrateCombine.pl.txt](https://github.com/sillsdev/TheCombine/files/5021383/migrateCombine.pl.txt)
(to run the script, remove the .txt extension and make sure it has execute permissions.  It is written in `perl` - standard on Ubuntu; available for Windows from [ActiveState](https://www.activestate.com/products/perl/) or [StrawberryPerl](http://strawberryperl.com/))
